src/scrap.py

```diff
@@ -0,0 +1,262 @@
+import requests
+import os
+import discord
+from typing import List
+from bs4 import BeautifulSoup
+from dotenv import load_dotenv
+import sqlite3
+
+# For using my .ENV files
+load_dotenv()
+
+
+# -------------------FOR SQLLITE----------------------------
+
+# https://docs.python.org/3/library/sqlite3.html
+# https://www.blog.pythonlibrary.org/2021/09/30/sqlite/#:~:text=Here%20is%20how%20you%20would%20create%20a%20SQLite,the%20sqlite3%20module%20will%20create%20an%20empty%20database.
+
+
+def create():
+    # Creates the DB. Only needs to be called once.
+
+    connection = sqlite3.connect("jobs.db")
+    cursor = connection.cursor()
+
+    cursor.execute(
+        """CREATE TABLE IF NOT EXISTS jobs(_company TEXT, _title TEXT, _location TEXT, _link TEXT, _processed BOOLEAN DEFAULT 0)"""
+    )
+
+    connection.commit()
+    connection.close()
```

Comment:
suggest using context manager instead of manually closing the connection
https://realpython.com/python-with-statement/

Reply:
+1 on Python's `with` syntax.

Reply:
@jaasonw Do you mind providing a suggestion including the usage of the `with` syntax in this situation?